package.json

```diff
@@ -27,10 +27,10 @@
   },
   "peerDependencies": {
     "chai": "^3.0.0 || ^4.0.0",
-    "cheerio": "0.19.x || 0.20.x || 0.22.x || 1.0.0-rc.1",
-    "enzyme": "1.x || ^2.3.0",
-    "react": "^0.14.0 || ^15.0.0-0",
-    "react-dom": "^0.14.0 || ^15.0.0-0"
+    "cheerio": "0.19.x || 0.20.x || 0.22.x || ^1.0.0-0",
+    "enzyme": "^2.9.1 || ^3.0.0",
```

Comment:
Out of curiosity, is it possible to support `"^2.7.0 || ^3.0.0"`?

Reply:
Yup. Done!